docs/content/projects/internal/BuildMetrics.md

```diff
@@ -27,31 +27,25 @@ They will be referred in docs as `<placeholder>`.
 
 [Http build cache](https://docs.gradle.org/current/userguide/build_cache.html#sec:build_cache_configure_remote) errors:
 
-- `<namespace>.build.cache.errors.[load|store].<http status code>`: errors counter
+- `<namespace>.<environment>.<node>.build.cache.errors.[load|store].<http status code>`: errors counter
```

Comment:
Added common prefix to build cache metrics.
These attributes are common and helpful for all build metrics.
Further decomposition can be done in specific metrics, but not required for others.